hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/ContextTransactionProcessor.java

```diff
@@ -110,7 +111,8 @@ public CallOutcome call() {
         // the error.
         final var hevmTransaction = safeCreateHevmTransaction();
         if (hevmTransaction.isException() && contractsConfig.chargeGasOnPreEvmException()) {
-            return chargeFeesAndReturnOutcome(hevmTransaction);
+            return maybeChargeFeesAndReturnOutcome(
+                    hevmTransaction, context.body().transactionIDOrThrow().accountIDOrThrow(), null, true);
```

Comment:
At this point it might be the case that the sender account is already known to the world updater - maybe?  If so then is it worth trying to look it up in case it's there, instead of just providing `null` as the sender account?

Reply:
At this point, we did not successfully create the HederaEvmTransaction object so it is not likely that the sender is good.  We only use the sender to return the sender nonce in the results which seems not relevant if we end up here.